packages/ui-react/src/components/Sidebar/__snapshots__/Sidebar.test.tsx.snap

```diff
@@ -1,104 +1,5 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`<SideBar /> > renders sideBar closed 1`] = `
-<div>
-  <div
-    class="_backdrop_577f70"
-  />
-  <div
-    class="_sidebar_577f70"
-    id="sidebar"
-    inert=""
-  >
-    <div
-      class="_heading_577f70"
-    >
-      <div
-        aria-label="close_menu"
-        class="_iconButton_0fef65"
-        role="button"
-        tabindex="0"
-      >
-        <svg
-          aria-hidden="true"
-          class="_icon_585b29"
-          viewBox="0 0 24 24"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <path
-            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
-          />
-          <path
-            d="M0 0h24v24H0z"
-            fill="none"
-          />
-        </svg>
-      </div>
-    </div>
-    <nav
-      class="_group_577f70"
-    >
-      <a
-        aria-current="page"
-        class="_button_f8f296 _default_f8f296 _outlined_f8f296 _active_f8f296"
-        href="/"
-      >
-        <span>
-          Home
-        </span>
-      </a>
-    </nav>
-  </div>
-</div>
-`;
+exports[`<SideBar /> > renders sideBar closed 1`] = `<div />`;
```

Comment:
Hm, all the content got removed. Is it expected?

Reply:
Yes, the Sidebar is now rendered in a modal/dialog which doesn't render when it is hidden.

Reply:
But... I found that the Sidebar open snapshot was also empty. This is because there is nothing rendered in the container due to the usage of portals in the Modal component. This is fixed now.